src/friendly-iframe-embed.js

```diff
@@ -90,6 +99,26 @@ export function setSrcdocSupportedForTesting(val) {
   srcdocSupported = val;
 }
 
+/**
+ * @param {!Window} win
+ * @return {function(*): !Promise<*>}
+ */
+function getDelayPromise(win) {
```

Comment:
IMHO this still should be called something different. Maybe `getDelayPromiseProducer()`?